### ISSUE TITLE
apiscript.js removes the ending punctuations.

### ISSUE THREAD

**Issue by GibreelAbdullah** [fawazahmed0/hadith-api#43]
Noticed that the database/line-by-line is missing all the ending punctuations in English translation (Didn't check them in others). I manually added them in one file (eng-malik) and regenerated the data using apiscript update command.

The manually added punctuation was removed when the updated file was created in line-by-line folder.

**Comment by fawazahmed0** [comment]
Yeah, actually the clean up is happening [here](https://github.com/fawazahmed0/hadith-api/blob/13a626be1d2a041305570c128c23a691c04e1ad7/utilities.js#L171), specifically [this line](https://github.com/fawazahmed0/hadith-api/blob/13a626be1d2a041305570c128c23a691c04e1ad7/utilities.js#L175) .
The above code was taken from [quran-api](https://github.com/fawazahmed0/quran-api#readme)

This could be fixed using [Unicode property escapes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Unicode_Property_Escapes) (i.e using something like `\P{P}` which will ignore puntuations).

Could you share me the file with punctuation?

**Comment by GibreelAbdullah** [comment]
I just added a full stop (.) at the end of hadith 1 of eng-malik to verify.

**Comment by GibreelAbdullah** [comment]
I would like to connect with you @fawazahmed0 . If you are ok kindly message me on telegram https://t.me/GibreelAbdullah

**Comment by fawazahmed0** [comment]
Brother, I don't regularly use telegram etc. But I do use my email regularly.

Anyways, the original files with puntuations, dots etc should be available at [here](https://github.com/fawazahmed0/hadith-api/tree/1/database/originals) and malik is [here](https://github.com/fawazahmed0/hadith-api/blob/1/database/originals/englishmalikscrapped.txt).

After fixing this issue the files needs to be regenerated and inshallah will look into this later.
